k8s.gcr.io/images/k8s-staging-dns/images.yaml

```diff
@@ -271,6 +281,7 @@
     "sha256:a507030925ea65b49e829611bc05c99b21d8333d690fc46e017ca1ab1bdc5dc6": ["1.22.0"]
     "sha256:1ab4f964a2ab7a06e220fbd0eea696ad60eec4d35552be459b030c48675ed656": ["1.22.1"]
     "sha256:ef44939a47d3a9451ba5b946496e0a40eb7f9c73a6e573a55b5a6dbfc577d97e": ["1.22.2"]
+    "sha256:47a7e7911397b220febcafdf8fbcb58619432de71d20f799ec9617e42f51c503": ["1.22.5"]
```

Comment:
could you also add hashed for sidecar container as well?

Reply:
yes, it is added in this PR